### PR TITLE
NXP 5.4.24-2.1.0 updates for EULA, imx-parser, alsa-lib

### PR DESCRIPTION
**Pull request by thochstein**

### CODE REVIEW

File: conf/machine/include/imx-base.inc [hunk 1]
```diff
@@ -158,7 +158,6 @@ MACHINE_SOCARCH_FILTER_append_imx = " \
 "
 MACHINE_SOCARCH_FILTER_append_imxvpu = " \
     imx-codec \
-    imx-parser \
```
Comment (otavio) [inline conf/machine/include/imx-base.inc:161]: @thochstein how so? could you explain why the change?

**Review by otavio — CHANGES_REQUESTED** [review]

File: conf/machine/include/imx-base.inc (hunk 1 above)
Comment (thochstein) [reply conf/machine/include/imx-base.inc:161]: There's nothing machine or SOC-specific in the recipe?
